spec/helpers/sessions_helper_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+# Specs in this file have access to a helper object that includes
+# the SessionsHelper. For example:
+#
+# describe SessionsHelper do
+#   describe "string concat" do
+#     it "concats two strings with spaces" do
+#       expect(helper.concat_strings("this","that")).to eq("this that")
+#     end
+#   end
+# end
+RSpec.describe SessionsHelper, :type => :helper do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
@RobPavl , when I receive mail notification about your 'completed' comment, I expect that when you say 'completed' on particular issue it is really completed. Issue is Completed when it is  fixed, tested, committed and pushed to github.
But now I'm confused - what does this comment mean?